### ISSUE TITLE
Front page switches to www-url after automatically awhile

### ISSUE THREAD

**Issue by developer-lindner** [polylang/polylang#81]
Hi,

i've encountered a fascinating issue on a client's website... after awhile (i guess due to regen of wp transients) the front page url switches to a www-url, every other page stays the same (non-www url).

I've used this awesome plugin several times now and never had an issue like that before with it.
My guess is that somehowt the issue is introduced by the server configuration, but i can't change it nor can i have a close look.

Has anyone else encountered such issues lately? Or is it polylang at the end itself?
Any help highly appreciated!

**Comment by grappler** [comment]
@developer-lindner What was the issue?

**Comment by developer-lindner** [comment]
@grappler The issue was that the system was getting requests with 'www' and somehow polylang used it after refreshing transients in the database. In `wp-config.php` some constants are set by using `$_SERVER['SERVER_NAME']`. Interestingly only the home page switched to 'www' later which was caused by the wrong value inside of the polylang transient. All other pages were still without it. I've modified the htaccess file to prevent https requests from having 'www' later (which is just a workaround and no real solution).
